contracts/transfer/build.rs

```diff
@@ -20,5 +29,20 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         keys_dir.to_str().unwrap()
     );
 
+    // Set the ID_[circuit_name] variables
+    let circuits = [
+        rusk_profile::Circuit::from_name("SendToContractTransparentCircuit"),
+        rusk_profile::Circuit::from_name("SendToContractObfuscatedCircuit"),
+        rusk_profile::Circuit::from_name("WithdrawFromTransparentCircuit"),
+        rusk_profile::Circuit::from_name("WithdrawFromObfuscatedCircuit"),
+        rusk_profile::Circuit::from_name("ExecuteCircuitOneTwo"),
+        rusk_profile::Circuit::from_name("ExecuteCircuitTwoTwo"),
+        rusk_profile::Circuit::from_name("ExecuteCircuitThreeTwo"),
+        rusk_profile::Circuit::from_name("ExecuteCircuitFourTwo"),
+    ];
+    for circuit in circuits {
+        set_id_env_var(&circuit.unwrap());
```

Comment:
Is it possibile to use `?` instead of `unwrap`? Would let problem easy to investigate